### ISSUE TITLE
Document or fix WML compatibility between different versions

### ISSUE THREAD

**Issue by Rondom** [filebench/filebench#81]
Hello again,
I am doing some work with evaluating the performance of FUSE and I am trying to reproduce the results of the paper ["Terra incognita: on the practicality of user-space file systems"](https://www.usenix.org/node/190584). The [workloads used are published](https://avatar.fsl.cs.sunysb.edu/wiki/projects/fuseperformancepublic/), as well.

While trying one of the workloads published in the link above, I noticed that 1.5 seems to breaks compatibility with older versions and I am wondering if this is on purpose (but not documented) or a bug.

    # ./filebench/filebench -f ./fuse_performance/fbench/32files-sread-1MB-32th.f
    Filebench Version 1.5-alpha1
    0.000: Allocated 173MB of shared memory
    0.000: Failed to open cvar directory around line 1
    0.000: syntax error at 'filesizegamma' around line 3
    Warning: no run command in the WML script!

With 1.4.9.1 it only seems to stumble over psrun

    ./filebench-1.4.9.1/filebench -f ./fuse_performance/fbench/32files-sread-1MB-32th.f
    Filebench Version 1.4.9.1
    IMPORTANT: Virtual address space randomization is enabled on this machine!
    It is highly recommended to disable randomization to provide stable Filebench runs.
    Echo 0 to /proc/sys/kernel/randomize_va_space file to disable the randomization.
    19928: 0.000: Allocated 170MB of shared memory
    19928: 0.000: Creating/pre-allocating files and filesets
    19928: 0.001: Fileset bigfiles: 32 files, 0 leafdirs, avg dir width = 32, avg dir depth = 1.0, 32768.000MB
    19928: 0.002: Removed any existing fileset bigfiles in 1 seconds
    19928: 0.002: making tree for filset /test/root/bigfiles
    19928: 0.002: Creating fileset bigfiles...
    19928: 59.228: Preallocated 32 of 32 of fileset bigfiles in 60 seconds
    19928: 59.228: waiting for fileset pre-allocation to finish
    19928: 59.228: Running 'sync'
    19928: 66.904: Running 'echo 3 > /proc/sys/vm/drop_caches'
    19928: 69.353: syntax error at 'psrun' on line 202

**Comment by sectorsize512** [comment]
Yes, we indeed are changing the grammar in 1.5. I'm in the process of rewriting the WML description page on github:

https://github.com/filebench/filebench/wiki/Workload-model-language

But it will take some time. We should be done with this before getting out of alpha releases.

I guess we will also need a page describing what was removed and added in 1.5 compared to 1.4.9. Assigning this issue  to myself. Related issue is #35.

In you specific case, *filesizegamma* attribute was removed. Instead, one has to use custom variables for *size* attribute. See webserver.f for an example.

Thanks!
